src/System.Drawing.Primitives/src/System/Drawing/Point.cs

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+
+namespace System.Drawing
+{
+    /// <devdoc>
+    ///    Represents an ordered pair of x and y coordinates that
+    ///    define a point in a two-dimensional plane.
+    /// </devdoc>
+    public struct Point
```

Comment:
Yes, creating a facade for 4.6 in the next commit. So will leave this as a new api add, and not attempt in this PR.